### ISSUE TITLE
Simple implementation question

### ISSUE THREAD

**Issue by cyberlp23** [ZulNs/Timepicker.js#2]
Is there anything else to make it work besides adding the few lines of code and uploading the js file? No css file? 
Because it doesn't seem to work on my page. What should I double check?

**Comment by ZulNs** [comment]
Have you convinced that your picker has been displayed by calling `timepicker.show()` method?

**Comment by cyberlp23** [comment]
You're right, it was missing, everything's fine now. Great script by the way! :)
One question though: there is the  timepicker.onPicked = function() , but is there a way to get time when the time changed but without having to click on the Pick button?
Thanks.

**Comment by ZulNs** [comment]
I plan to improve/add the feature you mean later, but it would take some time. I'll let you know when it's done.

**Comment by cyberlp23** [comment]
OK! Good to know.
One thing I don't get is how in your demo the clock is updated. Is there anything particular to trigger?

**Comment by ZulNs** [comment]
I will check it out as soon as possible when I have a spare time.

**Comment by cyberlp23** [comment]
Actually everything is fine, thanks again for the script!
